src/test/unit_tests/index.js

```diff
@@ -34,6 +34,7 @@ require('./test_sensitive_wrapper');
 // require('./test_debug_module');
 require('./test_range_stream');
 require('./test_buffer_pool');
+require('./test_s3select');
```

Comment:
It is under `// UTILS` I suggest moving it to`// CORE`.
Be aware that there 2 CORES sections, the second are for:

> Tests that require hosts pools